config/dev.exs

```diff
@@ -1,5 +1,6 @@
 use Mix.Config
 config :ethereumex,
+  url: System.get_env("ETHEREUM_URL"),
```

Comment:
@hayesgm Instead of adding another ```url``` option we can add ```url_path```  option so the Ethereum host will be configured: 
```"#{scheme()}://#{host()}#{port()}/#{url_path()}"```

the only thing that should be changed is ```port`` method
```
@spec port() :: integer()
  defp port do
    port = env_var(:port)
    if is_nil(port), do: "", else: port
  end

  @spec env_var(atom()) :: binary() | integer()| nil
  defp env_var!(var) do
    Application.fetch_env!(:ethereumex, var)
  end
end
```

What do you think?

Reply:
or we can remove ```scheme, host, port ``` options and just use ```url``` option

Reply:
Yeah, I personally prefer a URL option, since it's easier to pass in and it's easier to construct a URL from `scheme`, `host` and `port` instead of the reverse. This ends up being important because Infura gives you a full URL to access their nodes.

Reply:
Let me know if you want me to move forward with changing to a URL-only config.

Reply:
@hayesgm it's ok, let's use a URL-only config